plugins/include/sqlx.inc

```diff
@@ -127,149 +153,231 @@ native SQL_QuoteStringFmt(Handle:db, buffer[], buflen, const fmt[], any:...);
+ * @param query			The query string.
+ * @param data			Additional data array that will be passed to the handler function.
+ * @param dataSize		The size of the additional data array. 
  */
 native SQL_ThreadQuery(Handle:db_tuple, const handler[], const query[], const data[]="", dataSize=0);
 
 
 /**
- * Executes a query.
- * Returns 1 if the query succeeded.
- * Returns 0 if the query failed.
- * NOTE: You can call this multiple times as long as its parent
- *  connection is kept open.  Each time the result set will be freed
- *  from the previous call.
+ * Executes an already prepared query.
+ * 
+ * @note You can call this multiple times as long as its parent connection kept open. 
+ * 		 Each time the result set will be freed from the previous call.
+ *
+ * @param query			A handle of a prepared query to be executed.
+ * 
+ * @return 				1 if the query succeeded, 0 if the query failed.
  */
 native SQL_Execute(Handle:query);
 
+
 /**
  * Gets information about a failed query error.
- * Returns the errorcode.
+ *
+ * @param query			A handle of a query to extract the error from.
+ * @param error			A buffer where to store the error string.
+ * @param maxlength		The maximum length of the output buffer.
+ * 
+ * @return 				The error code.
  */
 native SQL_QueryError(Handle:query, error[], maxlength);
 
 
 /**
- * Returns 1 if there are more results to be read,
- *  0 otherwise.
+ * Check whether there are more results to be read.
+ *
+ * @param query			A handle of a query to check.
+ * 
+ * @return 				1 if there are more results, 0 otherwise.
  */
 native SQL_MoreResults(Handle:query);
 
 
 /**
- * Tells whether a specific column in the current row
- *  is NULL or not.
+ * Tells whether a specific column in the current row is NULL or not.
+ *
+ * @param query			A handle of a query to check.
+ * @param column		Which column to check for NULL.
+ * 
+ * @return 				1 if the column is NULL, 0 otherwise.
  */
 native SQL_IsNull(Handle:query, column);
 
+
 /**
  * Retrieves the current result.
- * A successful query starts at the first result,
- *  so you should not call SQL_NextRow() first.
- * Passing no extra params - return int
- * Passing one extra param - return float in 1st extra arg
- * Passing two extra params - return string in 1st arg, max length in 2nd
- * Example:
- *  new num = SQL_ReadResult(query, 0)
- *  new Float:num2
- *  new str[32]
- *  SQL_ReadResult(query, 1, num2)
- *  SQL_ReadResult(query, 2, str, 31)
+ * A successful query starts at the first result, so you should not call
+ * SQL_NextRow() first.
+ * 
+ * @note Example how to get different types of values:
+ *  	  new num = SQL_ReadResult(query, 0)
+ *  	  new Float:num2
+ *  	  new string[32]
+ *  	  SQL_ReadResult(query, 1, num2)
+ *  	  SQL_ReadResult(query, 2, str, 31)
+ *
+ * @param query			A handle of a query to read results from.
+ * @param column		Which column to get the value from.
+ * @param ...			Passing no extra arguments - returns an integer.
```

Comment:
Same as I noted above. When you just reuse the old formatting you even have inconsistencies within this file. You should pay attention to that.